Extension/src/common.ts

```diff
@@ -175,7 +175,15 @@ export function resolveVariables(input: string, additionalEnvironment: {[key: st
             }
             case "config": {
                 let config: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration();
-                newValue = (config) ? config.get(name).toString() : undefined;
+                try {
```

Comment:
This seems backwards to me. The previous case first did the iteration over the key array and threw an exception (the bug we're trying to fix). So, we should try that first and if we throw, then we fall back to the direct setting.

Reply:
It doesn't make sense to try the code that we expect to fail and have no repro which actually works. I think the VS Code API changed. I just added that try/catch in case there's some reason the single get might fail that we're not aware of.

Reply:
But what this means is that if there IS any behavior that we haven't been able to discover, then that behavior will break. If we do the opposite, that won't occur. It is a good point that we shouldn't regularly throw in a common code-path, so is there a way to do the same thing without relying on an exception?

Reply:
The old code no longer matches the VS Code APIs used so it doesn't make sense to fall back to it. Either it never worked or the API/types changed.